.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -33,24 +33,28 @@ assignees: ''
 - [ ] Merge PR
 - [ ] Create and push *both* tags to GitHub (`vX.Y.Z`, `X.Y.Z`)
   - Pull latest branch locally and run `contrib/release/tag-release.sh`
-- [ ] Ask a maintainer to approve the build in the following links:
+- [ ] Ask a maintainer to approve the build in the following links (keep the URL
+      of the GitHub run to be used later):
   - [Cilium v1.10](https://github.com/cilium/cilium/actions?query=workflow:%22Image+Release+Build%22)
   - [Cilium v1.9](https://github.com/cilium/cilium/actions?query=workflow:%22Image+Release+Build+v1.9%22)
   - [Cilium v1.8](https://github.com/cilium/cilium/actions?query=workflow:%22Image+Release+Build+v1.8%22)
   - [Cilium v1.7](https://github.com/cilium/cilium/actions?query=workflow:%22Image+Release+Build+v1.7%22)
   - Check if all docker images are available before announcing the release
     `make -C install/kubernetes/ check-docker-images`
-- [ ] Get the image digests from the build process (click in the GH Run > "Display Digests"),
-      update the digests in the helm charts locally and make a commit / PR with the digests.
+- [ ] Get the image digests from the build process and make a commit / PR with
+      the digests.
+    - [ ] Run `contrib/release/pull-docker-manifests.sh` to fetch the image
+          digests, use the URL of the GitHub run here.
 - [ ] Create helm charts artifacts in [Cilium charts] repository using
       [cilium helm release tool] for both the `vX.Y.Z` release and `vX.Y` branch
-      & push to repository
+      and push these changes into the helm repository as well as make a commit
+      and push them into cilium repository.
```

Comment:
nit: At the moment it's not *so* obvious that the pull-docker-manifests is actually generating different files in the tree and those are the changes that need to be created into the new commit.

Same content, minor reorganisation.

```suggestion
- [ ] Update the Helm charts in the [Cilium charts] repository and the main repository
  - [ ] Run `contrib/release/pull-docker-manifests.sh` in the Cilium repository to fetch the image digests, use the URL of the GitHub run here.
  - [ ] Run the [cilium helm release tool] from the cilium-charts repository to generate the `vX.Y.Z` release charts and `vX.Y` branch charts.
  - [ ] Push the new helm charts to the Cilium charts repository.
  - [ ] After pushing the charts, add the local changes from the Cilium repository to a new commit and open a PR against the release branch to update the digests and charts.
```

Reply:
Discussed offline, it should be something around:
```
- [ ] Commit and merge into cilium/cilium
- [ ] Fetch the branch
- [ ] Create helm charts artifacts in [Cilium charts] repository using
      [cilium helm release tool] for both the `vX.Y.Z` release and `vX.Y` branch
      and push these changes into the helm repository.
```